src/content_cao.h

```diff
@@ -201,6 +201,8 @@ class GenericCAO : public ClientActiveObject
 			float time_from_last_punch=1000000);
 
 	std::string debugInfoText();
```

Comment:
there is a bugfix to do there. debugInfoText is virtual in parent class. Please set it virtual too